backend/app/core/langfuse/langfuse.py

```
@@ -1,10 +1,12 @@
 import uuid
 import logging
-from typing import Any, Dict, Optional
+from typing import Any, Callable, Dict, Optional
```

Ruff (0.14.6)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)